lms/resources/_js_config/__init__.py

```diff
@@ -203,6 +209,10 @@ def enable_lti_launch_mode(self, course, assignment: Assignment):
         }
         self._config["debug"]["values"] = self._get_lti_launch_debug_values()
 
+        document = self._request.find_service(
+            DocumentService
+        ).get_document_from_assignment(assignment)
+        self._config["document"] = {"filename": document.filename}
```

Comment:
This value might also be useful for editing (to display something better than "A canvas file")

We could read it form here or also add it to the editing dict below?